module6/lessons/Week1/ToolsForAfterAnInterview.md

```diff
@@ -0,0 +1,62 @@
+
+Read through the blog post and then reflect on the following questions:
+
+What parts of this blog post resonate with you?
+
+How does a vulnerability hangover show up for you?
+
+What are some ways that you can nurse your vulnerability hangover so you can continue to be brave? What’s your "soft landing"?
+
+Are there ways that your group can support you in your "soft landing"? For example, maybe there is something they can remind you to do for yourself or a mantra they can remind you of.
+
+When we come back together after today's interviews, you will discuss these questions with your group and then I'll ask for volunteers to share in the full group. Then we will move into an Interview Recap and Emotional Check-In for today's interview.
+
+### Interview Recaps
+
+After finishing an interview, no matter how the interview went, it's key to reflect on the experience and grow from it.
+
+Start by taking time to individually reflect on the following three questions.
+
+1. What went well in this interview?
+2. What could have gone better?
+3. What steps do you want to take to better prepare for your next interview?
+
+Then, in your group, take turns sharing your responses to these questions. Everyone processes differently, so it's normal if one person prefers to share briefly and the next chooses to share more and do more external processing.
+
+### Emotions Check In
+
+Interviews are both a logical and an emotional activity. Once you check in about the logical part of the interview and what you want to improve on as a result, it's also important to check in emotionally with where you are and create a "soft landing" for yourself.
+
+Start by taking time to individually reflect on the following two questions.
+
+1. How are you emotionally right now? (Check in for self-doubt, insecurity, fatigue, cravings, wanting to hide)
+2. If you are having any vulnerability hangover symptoms, what can you do to create a "soft landing" and take care of yourself? (Think about if there is anything your group or other support people in your life can help you with here. You don't have to build your soft landing alone.)
+
+Then, in your group, take turns sharing as much as you're comfortable. Everyone processes differently, so it's normal if one person prefers to share briefly and the next chooses to share more and do more external processing. That being said, I encourage you to push yourself in sharing. Many people find that tough emotions have less power when they are named and shared.
+
+If you're a listener, remember that your job is to listen and maybe share empathy. Hold off from jumping in with solutions or ideas unless you're explicitly asked for that.
+
+### Conclusion
+
+We will give space for groups to do Interview Recaps and Emotions Check SIns after each Friday interview. We encourage you to also find time to support each other after your real interviews. 

```

Comment:
This is reading like 'check SINS' 😆 

Also, Friday should be Thursday

Reply:
Good catches!